presto-main/src/main/java/com/facebook/presto/server/TaskUpdateRequest.java

```diff
@@ -38,6 +43,7 @@
     private final List<TaskSource> sources;
     private final OutputBuffers outputIds;
     private final Optional<TableWriteInfo> tableWriteInfo;
+    private final Optional<Codec<PlanFragment>> fragmentCodec;
```

Comment:
Add a comment this is only required on coordinator side when serializing `PlanFragment`, not provided on worker :)

Reply:
See: https://github.com/prestodb/presto/compare/6b5757b8ed4fb19aee6c257aec17a21cd16c068f..e70403ed7c21ca5e8206208db1b056d2f3137fca